src/ocm/components/clusterConfiguration/networkConfiguration/AdvancedNetworkFields.tsx

```diff
@@ -67,6 +67,7 @@ const AdvancedNetworkFields = () => {
                     isRequired
                     labelInfo={index === 0 && isDualStack ? 'Primary' : ''}
                   />
+                  <br />
```

Comment:
This problem of missing separation is in many of the fields following this one.

PTAL at this cluster http://edge-13.edge.lab.eng.rdu2.redhat.com:6008/clusters/682dfdff-54cd-4217-9eca-ffb929649f23
and see if we can fix them all.

Couldn't we use `padding` instead?

Reply:
@celdrake  Something like this?
![image](https://user-images.githubusercontent.com/11390125/201045390-025d640f-2cc2-4acb-b309-addb1509b329.png)

Reply:
![image](https://user-images.githubusercontent.com/11390125/201062188-2f906eee-3ada-45c5-be68-9f6fb6f96f16.png)

Reply:
Yes, please let's fix all those fields too as in your screenshot.